### PR TITLE
Add Petra bug bounty

### PR DESCRIPTION
**Pull request by kent-white**
We are moving the bug bounty to github

### CODE REVIEW

File: bug_bounty.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Duplicate Reports
+
+Compensation for duplicate reports will be split among reporters with the first to report taking priority using the following equation:
+
+```
+R: total reports
+ri: report priority
+bi: bounty share
+
+bi = 2 ^ (R - ri) / ((2^R) - 1)
```
Comment (hariria) [inline bug_bounty.md:60]: $$b_i = \frac{2^{R - r_i}}{2^R - 1}$$

Comment (hariria) [reply bug_bounty.md:60]: We could add latex straight in so it's clearer?

**Review by hariria — APPROVED** [review]
Nice

File: bug_bounty.md (hunk 1 above)
Comment (kent-white) [reply bug_bounty.md:60]: will do

**Pull request merged**